src/Stack/index.jsx

```diff
@@ -47,6 +57,7 @@ Stack.defaultProps = {
   direction: 'vertical',
   gap: 0,
   className: undefined,
+  reverse: false,
```

Comment:
We need to add an example for this property. Also we need to add a test for this new property.